static/src/javascripts/projects/common/modules/experiments/affix.js

```diff
@@ -30,15 +34,17 @@ define([
     Affix.CLASSY_BOTTOM = 'affix-bottom';
 
     Affix.prototype.calculateContainerPositioning = function () {
-
         // The container defines the static positioning of the affix element.
-        this.$container.css('top', '0');
-        var containerTop = this.$markerTop.offset().top - this.$container.offset().top;
-        this.$container.css('top', containerTop + 'px');
-    };
-
-    Affix.prototype.checkPositionWithEventLoop = function () {
-        requestAnimationFrame(this.checkPosition.bind(this));
+        var that = this;
+        fastdom.write(function () {
+            that.$container.css('top', '0');
+            fastdom.read(function () {
+                var containerTop = that.$markerTop.offset().top - that.$container.offset().top;
+                fastdom.write(function () {
+                    that.$container.css('top', containerTop + 'px');
```

Comment:
Callback hell redux? :grin: 

Considering that `calculateContainerPositioning` is called inside a fastdom.write: `write -> write -> read -> write`.
I'd give an extra round of testing to this method. :wink: